changelog/fix_lint_non_actomic_file_operation_in_case_of_implicit_receiver.md

```diff
@@ -0,0 +1 @@
+* [#13558](https://github.com/rubocop/rubocop/pull/13558): Fix `Lint/NonAtomicFileOperation` failure in case of implicit receiver. ([@viralpraxis][])
```

Comment:
That's just my two cents. "error" seems more appropriate than "failure" in terms of vocabulary.

Reply:
I agree, thanks.